transport_plugins/bled112/iotile_transport_bled112/bled112_cmd.py

```diff
@@ -78,6 +83,30 @@ def run(self):
                 self._logger.exception("Fatal error in background processing loop")
                 raise
 
+    def update_conn_map(self, conn_string, device_uuid):
```

Comment:
Same comment as above, this behavior can stay inside of `BLED112DeviceAdapter` and it should simplify the code.

Reply:
Thanks for the suggestion. Implemented in this latest push.